### PR TITLE
[tvOS] Improve tvOS code based upon comments.

### PR DESCRIPTION
**Pull request by aaronschubert0**
@appleguy Here are my improvements based upon your comments. Feels a lot better now.

### CODE REVIEW

**Comment by appleguy** [comment]
@aaronschubert0 no idea how this already has a conflict!  I've hardly merged anything...

**Comment by aaronschubert0** [comment]
@appleguy Huh weird, yeah was fine this morning. No worries, will fix tomorrow morning.

**Comment by aaronschubert0** [comment]
@appleguy All good now :)

File: AsyncDisplayKit/ASControlNode+tvOS.m [hunk 1]
```diff
@@ -0,0 +1,88 @@
+//
+//  ASControlNode+tvOS.m
+//  AsyncDisplayKit
+//
+//  Created by Aaron Schubert on 21/04/2016.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+#if TARGET_OS_TV
```
Comment (appleguy) [inline AsyncDisplayKit/ASControlNode+tvOS.m]: These files are great!  Scalable pattern that will allow us to support some cool stuff.

**Pull request merged**